### PR TITLE
Fix use of accent colors for hosts that don't support them

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

For existing hosts that don't expose the new Accent colors, need to default to a different color.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [x] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs [hunk 1]
```diff
@@ -759,7 +759,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
...
                                 $output = [System.Text.StringBuilder]::new()
                                 $prefix = ' ' * $indent
-                                $accentColor = Get-VT100Color $Host.PrivateData.FormatAccentColor
+
+                                if ($null -ne $Host.PrivateData.FormatAccentColor) {
+                                    $accentColor = Get-VT100Color $Host.PrivateData.FormatAccentColor
+                                }
+                                else {
+                                    $accentColor = Get-VT100Color [Console]::ForegroundColor
+                                }
```
Comment (TylerLeonhardt) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: perfect time to use the ternary op?

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs [hunk 2]
```diff
@@ -1006,7 +1013,13 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
...
                                         if ($Host.PrivateData) {
                                             $errorColor = Get-VT100Color -color $Host.PrivateData.ErrorForegroundColor
-                                            $accentColor = Get-VT100Color -color $Host.PrivateData.ErrorAccentColor
+
+                                            if ($null -ne $Host.PrivateData.ErrorAccentColor) {
+                                                $accentColor = Get-VT100Color -color $Host.PrivateData.ErrorAccentColor
+                                            }
+                                            else {
+                                                $accentColor = $errorColor
+                                            }
```
Comment (TylerLeonhardt) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: perfect time to use the ternary op?

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs (hunk 1 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: null coalescing is even better here

Comment (SteveL-MSFT) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: I tried a version with both ternary and null coalescing.  Works but hard to read.  Going to not use ternary for this case.

**Review by TylerLeonhardt — APPROVED** [review]
LGTM

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs (hunk 1 above)
Comment (TylerLeonhardt) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: Still on that future syntax 😎

**Review by anmenaga — APPROVED** [review]

**Pull request merged**